### DISCUSSION TITLE
Unable to view CSS on my site

### DISCUSSION THREAD

**Discussion by caretakerx** [vercel/vercel#6914] [Help]
I'm not able to view my css on my Vercel site.

**Comment by caretakerx** [comment]
Any help.

**Reply by leerob** [reply]
Could you provide more details? What isn't working, what have you tried? Is there a link to code or the repository?

**Comment by caretakerx** [comment]
I hosted my site on Vercel on Visual Studio Code and the CSS doesn’t display. I checked the source code and the CSS is there and uploaded.

<html>
<head>
<title>Brett O'Connor Designs</title>
<meta charset="UTF-8">
<meta name="format-detection" content="telephone=no">
<meta name="viewport" content="width=device-width, inititial-scale=1.0, maximum-scale=1.0, user-scalable=0">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<link rel="stylesheet" type="text/css" href="css/style.css">

**Reply by delbaoliveira** [reply]
Hi @caretakerx,

Assuming the link in your profile is your website, in your CSS file, there is a rule that looks like invalid CSS which is causing all styles below it not to work. The `abbr` is missing a closing square bracket. 

```css
abbr[title::after {
    content: " (" attr(title) ")";
}
```
	
Adding the bracket fixed the styles on my local reproduction.

**Comment by caretakerx** [comment]
I fixed the error but now my vercel —prod command is showing an error. Error! request to https://api.vercel.com/v13/now/deployments failed, reason: socket hang up

**Reply by delbaoliveira** [reply]
Would you be able to try again? 🙏🏼 
It worked with my local reproduction so it may have been related to a temporary issue.

**Comment by caretakerx** [comment]
I’m not able to get my vercel command to work to host the site.

**Comment by caretakerx** [comment]
Do you know the vercel commands for hosting an app.
